### PR TITLE
OPSEXP-1549: add tomcat10 images

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-1549

### CODE REVIEW

**Review by gionn — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -33,22 +35,24 @@ RUN \
...
+    curl -fsSLo tcnative${filetype}  ${active_mirror}/tomcat/tomcat-connectors/native/${TCNATIVE_VERSION}/source/tomcat-native-${TCNATIVE_VERSION}-src.tar.gz; \
   done; \
   \
   echo "$TOMCAT_SHA512 *tomcat.tar.gz" | sha512sum -c - || (echo "Checksum did't match: $(sha512sum *tomcat.tar.gz)" && exit 1); \
+  echo "$TCNATIVE_SHA512 *tcnative.tar.gz" | sha512sum -c - || (echo "Checksum did't match: $(sha512sum *tcnative.tar.gz)" && exit 1); \
   \
-  gpg --batch --verify tomcat.tar.gz.asc tomcat.tar.gz && \
-  tar -zxf tomcat.tar.gz -C /build/tomcat --strip-components=1
+  #gpg --batch --verify tcnative.tar.gz.asc tcnative.tar.gz && \
+  #gpg --batch --verify tomcat.tar.gz.asc tomcat.tar.gz && \
```
[the remark this answers is not in the document]
Comment (alxgomz) [reply Dockerfile]: totally!

File: updatecli.d/01_tomcat_version.tpl [hunk 2]
```diff
@@ -1,27 +1,47 @@
...
+name: "Bump tomcat versions"
 
 scms:
   tomcatGitHub:
     kind: "git"
     spec:
-        url: "https://github.com/apache/tomcat.git"
-        branch: "main"
+      url: "https://github.com/apache/tomcat.git"
+      branch: "main"
```
Comment (gionn) [inline updatecli.d/01_tomcat_version.tpl:8]:
```suggestion
      url: https://github.com/apache/tomcat.git
      branch: main
```

consistency

File: updatecli.d/01_tomcat_version.tpl [hunk 3]
```diff
@@ -1,27 +1,47 @@
...
+      file: tomcat{{ requiredEnv "TOMCAT_MAJOR" }}.json
+      key: tomcat_version
+  tcnativeJson:
     name: Update version in json target
     kind: json
-    sourceid: github
+    sourceid: tcnativeTag
     spec:
       file: tomcat{{ requiredEnv "TOMCAT_MAJOR" }}.json
       key: tomcat_version
```
Comment (gionn) [inline updatecli.d/01_tomcat_version.tpl:47]:
```suggestion
      key: tcnative_version
```

File: updatecli.d/02_tomcat_checksum.tpl [hunk 4]
```diff
@@ -1,19 +1,33 @@
...
+  tomcatChecksum:
+    name: Retrieve the tomcat binaries archive checksum
     kind: shell
     spec:
-      command: ./get-checksum.sh
+      command: ./get-checksum.sh tomcat
+      environments:
+        - name: TOMCAT_MAJOR
+  tcnativeChecksum:
+    name: Retrieve the tomcat native libs  checksum
```
Comment (gionn) [inline updatecli.d/02_tomcat_checksum.tpl:12]:
```suggestion
    name: Retrieve the tomcat native libs checksum
```

**Review by gionn — CHANGES_REQUESTED** [review]

**Review by gionn — APPROVED** [review]

**Pull request merged**